hooks/useGetTVL.ts

```diff
@@ -0,0 +1,30 @@
+import useSynthetixQueries from '@synthetixio/queries';
+import { wei } from '@synthetixio/wei';
+import { useQuery } from 'react-query';
+import Connector from '../containers/Connector';
+import { useStakedSNX } from './useStakedSNX';
+
+export const useGetTVL = () => {
+  const { isL2 } = Connector.useContainer();
+  const stakedSnxQuery = useStakedSNX();
+
+  const { useExchangeRatesQuery } = useSynthetixQueries();
+  const { data: exchangeRateData } = useExchangeRatesQuery();
+
+  return useQuery(
```

Comment:
Could this hook be a candidate for [useQueries](https://tanstack.com/query/v4/docs/reference/useQueries?from=reactQueryV3&original=https://react-query-v3.tanstack.com/reference/useQueries)

Would reduce the loading/data states to 1

Reply:
hmm it would be very nice, but we're composing queries and [useQueries](https://tanstack.com/query/v4/docs/reference/useQueries?from=reactQueryV3&original=https://react-query-v3.tanstack.com/reference/useQueries)` expect you to pass functions. (Or objects with queryKey and functions)